### PR TITLE
Update U2F App ID guidance in documentation

### PR DESCRIPTION
**Pull request by codingllama**
Update U2F documentation to not include port 443 in the U2F App ID.

This is recommended by the standard, plus including port 443 in the App ID causes issues with Chrome, as explained at https://github.com/gravitational/teleport/issues/8017#issuecomment-931594038. Non-standard ports are fair game and should be included.

References:
* https://fidoalliance.org/specs/fido-u2f-v1.0-ps-20141009/fido-appid-and-facets-ps-20141009.html#determining-the-facetid-of-a-calling-application
* https://developers.yubico.com/U2F/App_ID.html
* https://www.npmjs.com/package/u2f-api#user-content-common-problems

### CODE REVIEW

**Comment by codingllama** [comment]
I intend to backport this to v7 once approved, let me know if you'd like me to push it even further back.

**Comment by codingllama** [comment]
Fixes https://github.com/gravitational/teleport/issues/8017.

**Review by klizhentas — APPROVED** [review]

**Review by zmb3 — APPROVED** [review]

**Comment by codingllama** [comment]
Thanks everyone for the quick review.

**Pull request merged**
